Jenkinsfile

```diff
@@ -31,6 +42,7 @@ node {
         }
       } finally {
         sendhipchat()
+        sh "./acceptance-tests/runner/scripts/stop-selenium.sh"
```

Comment:
*really* want to make sure it stops this hey ;) this seems the right place... I need to compare with ATH jenkinsfile